src/v2/view-builder/views/end-user-remediation/EndUserRemediationMessages.js

```diff
@@ -64,16 +66,16 @@ export default View.extend({
     </div>
   `,
   getTemplateData() {
-    const messages = this.options.messages.value;
+    const messages = this.options.messages;
```

Comment:
Does this change affect other views besides the `DeviceAssuranceGracePeriodView`?

Reply:
Yeah it affects the end user remediation error terminal view, but I refactored that to accommodate this change. I'll tag you at that change for viz

Reply:
Okay great, as long as we're updating all of the places this affects